doc/unlocking_the_bootloader.md

```diff
@@ -1,6 +1,10 @@
 #### On unlocking the bootloader
-Devices by *Samsung*, *Google* and *Fairphone* make it fairly easy to unlock the bootloader and receive good support in the installer.
+Devices by *Samsung*, *Google*, *OnePlus* and *Fairphone* make it fairly easy to unlock the bootloader and receive good support in the installer.
 
-Some devices with require manual steps to unlock the bootloader. In general you will need to create an account at a vendor website and receive some code from there. OpenAndroidInstaller will try to guide you as far as possible. These vendors include *Sony, Motorola, Xiaomi* and *OnePlus* among others.
+Some devices with require manual steps to unlock the bootloader. In general you will need to create an account at a vendor website and receive some code from there. OpenAndroidInstaller will try to guide you as far as possible. These vendors include *Sony, Motorola, Xiaomi*  among others.
 
-Other phone vendors stops allowing to unlock the bootloader all together. There is nothing to be done if you didn't unlock your device in time. These vendors include *Huawei and LG* among others. Support for these vendors will always be very limited.
+Other phone vendors stops allowing to unlock the bootloader all together. There is nothing to be done if you didn't unlock your device in time. These vendors include *Huawei  LG Asus* among others. Support for these vendors will always be very limited.
```

Comment:
Could you please separate Huawei, LG and Asus with a comma and _and_? :D

Reply:
I removed all the "and" so it's simpler to edit